src/internal/operators/publish.ts

```diff
@@ -42,23 +42,23 @@ export function publish<T, O extends ObservableInput<any>>(selector: (shared: Ob
  * ![](publish.png)
  *
  * ## Examples
- * Make source$ hot by applying publish operator, then merge each inner observable into a single one
- * and subscribe.
+ *
+ * Make `source$` hot by applying `publish` operator, then merge each inner observable into a single one
+ * and subscribe
```

Comment:
Most examples don't have dot (`.`) at the end of a sentence, so I removed it wherever there was one. I hate it, I'd rather have it...